plugin.video.arloview/resources/lib/monotonic.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
```

Comment:
This shoule be split into a script.module.

Reply:
This is a separate module, not sure what you are asking me to do...